bin/apt-dependencies.sh

```diff
@@ -118,8 +118,8 @@ EOF
 fi
 
 # rest of python dependencies
-if [ "${VERSION_CODENAME}" == "bookworm" ]; then
-    # On Debian bookworm, need the --break-system-package to into to default system location
+if [ "${VERSION_CODENAME}" == "bookworm" -o "${VERSION_CODENAME}" == "noble" ]; then
+    # On Debian bookworm and Ubuntu noble, need the --break-system-package to into to default system location
     pip3 --default-timeout=10000 install --break-system-packages --upgrade sphinx_rtd_theme nose requests hypothesis==3.79.0
 else
     pip3 --default-timeout=10000 install --upgrade sphinx_rtd_theme nose requests hypothesis==3.79.0
```

Comment:
s.a.

Reply:
changed it, the packages are already available, so no need to break anything

Reply:
Oh sorry, I meant the pip3 lines, we need those only in Mango and Docs, but they create their own venv iirc…